src/backend/overviewSite/index.html

```diff
@@ -152,80 +152,74 @@ <h1>Open Event Web Apps</h1>
 
 <div class="container-fluid bg-3">
     <h2 class="margin"> Apps with expandable sessions page </h2>
-    <div class="row">
-        <div class="col-md-4 col-sm-4 col-xs-12 margin">
+    <div class="row text-center">
+        <div class="col-md-4 col-sm-6 col-xs-12 margin">
             <p><strong>Open Tech Summit 2017</strong></p>
             <a href='./OpenTechSummit/index.html' target='_blank'>
                 <img src="./otssmall.jpg" class="" alt="Image">
             </a>
         </div>
-        <div class="col-md-4 col-sm-4 col-xs-12 margin">
+        <div class="col-md-4 col-sm-6 col-xs-12 margin">
             <p><strong>Facebook Developer Conference 2017</strong></p>
             <a href='./F8-FacebookDeveloperConference2017/index.html' target='_blank'>
                 <img src="./fbsmall.jpg" class="" alt="Image">
             </a>
         </div>
-        <div class="col-md-4 col-sm-4 col-xs-12 margin">
+        <div class="col-md-4 col-sm-6 col-xs-12 margin">
             <p><strong>FOSSASIA Summit 2017</strong></p>
             <a href='./FOSSASIASummit/index.html' target='_blank'>
                 <img src="./fasmall.jpg" class="" alt="Image">
             </a>
         </div>
-    </div>
-    <div class="row">
```

Comment:
@abishekvashok Just one thing do not remove the rows. It is semantically incorrect. Good Job!

Reply:
@geekyd basically it would split it up, i.e, when it shows 3 rows in large screens it would be pushed down to bottom on tablet, so it will look orphaned like this:

![screenshot_20180110_045224](https://user-images.githubusercontent.com/8947010/34775021-b0e4a26a-f637-11e7-91f6-0a801b2d0c83.png)

(I have'nt removed the seperation btw. one page events and other events)